inference/core/roboflow_api.py

```diff
@@ -124,7 +124,7 @@ def add_custom_metadata(
     inference_ids: List[str],
     field_name: str,
     field_value: str,
-) -> bool:
+) -> None:
```

Comment:
Every method would return `None`, I'd vote for removing `-> None` annotation

Reply:
👍